lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -437,8 +437,14 @@ private function getTableDiffForAlterForeignKey(ForeignKeyConstraint $foreignKey
 
     private function parseColumnCollationFromSQL(string $column, string $sql) : ?string
     {
-        $pattern = '{(?:\W' . preg_quote($column) . '\W|\W' . preg_quote($this->_platform->quoteSingleIdentifier($column))
-                 . '\W)[^,(]+(?:\([^()]+\)[^,]*)?(?:(?:DEFAULT|CHECK)\s*(?:\(.*?\))?[^,]*)*COLLATE\s+["\']?([^\s,"\')]+)}isx';
+        if (PHP_VERSION_ID < 70300) {
```

Comment:
Can you please add comment with a link to the bug report explaining this condition? Thanks.
https://bugs.php.net/bug.php?id=75355

Reply:
done